database/src/main/resources/db/migration/V1_77__pre_notice_comments.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE pre_notice_comment (
+  id SERIAL PRIMARY KEY,
+  "pre-notice-id" INTEGER REFERENCES pre_notice (id),
+  "created-by" TEXT REFERENCES "user" (id),
+  created TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+  "comment" TEXT
```

Comment:
No need to use quotation marks

Reply:
I thought that it was a reserved word, but it's not. Good catch.